lib/client.js

```diff
@@ -311,7 +341,48 @@ Client.prototype = {
     }.bind(this));
   },
 
+  /// #### client.instance(instanceGuid)
+  ///
+  /// Create an Client wrapper for another app instance.
+  ///
+  /// ##### Arguments
+  ///
+  ///   * `instanceGuid` the GUID of the other instance of your app
+  ///
+  /// ##### Returns
+  ///
+  /// A [Promises/A+](https://promisesaplus.com) conformant `promise` object.
+  ///
+  /// ```javascript
+  /// var client = ZAFClient.init();
+  ///
+  /// client.get('instances').then(function(data) {
+  ///   var guid = Object.keys(data.instances).filter(function(guid) {
```

Comment:
probably because I just hada discussion on APIs and associative arrays, but I think objects as arrays is fine for internal mechanism, but terrible for public APIs.